src/frontend/src/components/Case/index.tsx

```diff
@@ -1,30 +1,40 @@
 import React from 'react';
 import { CaseProps } from '../../redux/records/types';
+import Charges from '../Charges';
 
 export default class Cases extends React.Component<CaseProps> {
   render() {
-    const { name, case_number, birth_year, balance_due } = this.props.case;
+    const {
+      name,
+      case_number,
```

Comment:
Field names are set by the JSON response. Maybe ticket this for now and we can discuss during the meetup if we want to take care of it using on the frontend with something like `https://lodash.com/docs/4.17.15#camelCase` or on the backend?

Reply:
This also affects https://github.com/codeforpdx/recordexpungPDX/pull/557

Reply:
@KentShikama I won't be at the meeting this week so if you think it's best to just ticket this, that's fine with me. 

I'm aware these are the field names from the JSON responses. The correct way to handle this is probably to have constructors for these objects defined next to their types. This could be plain Javascript:

    createCaseType(input) {
      return {
        balanceDue: input.balance_due
        birthYear: input.birth_year
        ...
      }
    }

Or a Typescript class:

    class Greeter {
      balanceDue: number;
      birthYear: number; 

      constructor(input: Object) {
        this.balanceDue = input.balance_due;
        this.birthYear = input.birth_year;
        ...
      }
    }

This code is straightforward to write and maintain. 

I have also used https://github.com/domchristie/humps in the past which worked well. IMO the backend should not send responses using camelCase by default because APIs generally shouldn't know that much about how they're being used.